lib/loader/index.js

```diff
@@ -235,6 +318,13 @@ module.exports = {
 
       try {
         injectionNode.appendChild(link);
+
+        if (options.namespace) {
```

Comment:
You could probably simplify these sorts of if-block by assigning loadedUrls to namespace (and reassign when clearing), so you'd always be able to check the local variable.